espanol/cielo/cielo.md

```diff
@@ -1,6 +1,9 @@
 
 Muchos soldados quedaron petrificados, pensaban que era el fin del mundo.., 
 otros pensaron que los enemigos habian creado una nueva arma para confundirlos,
-cuando el capitán pudo percatarse se dió cuenta que:
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.